src/__/collections/has.php

```diff
@@ -29,6 +29,12 @@ function has($collection, $path)
 //         $has = \__::isObject($collection) ? 'property_exists' : 'array_key_exists';
```

Comment:
I think you can remove this commented code. (Code before the _numeric keys for objects_ issue was found, see comment below)

It certainly won't be used soon.

Reply:
Removed 👍